src/components/DayPickerKeyboardShortcuts.jsx

```diff
@@ -33,15 +33,91 @@ const defaultProps = {
   phrases: DayPickerKeyboardShortcutsPhrases,
 };
 
+function getKeyboardShortcuts(phrases) {
+  return [{
+    unicode: '↵',
+    label: phrases.enterKey,
+    action: phrases.selectFocusedDate,
+  },
```

Comment:
the indentation here is a little off; if it's going to start with `[{`, then each line needs to be `}, {` - otherwise the first `{` needs to be on its own line.